### PR TITLE
Fix wrong header level in changelog

### PR DESCRIPTION
**Pull request by kianmeng**
Before:
image

After:
image

### CODE REVIEW

**Comment by SteffenDE** [comment]
Hi there, thank you for the PR. I'm not sure if the first two headings shouldn't actually be left as is. Having the Introduction of verified routes and the phx.new revamp at the top of the navigation is probably fine.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ See the [upgrade guide](https://gist.github.com/chrismccord/00a6ea2a96bc57df0cce
 
 Phoenix v1.7 requires Elixir v1.11+ & Erlang v22.1+.
 
-## Introduction of Verified Routes
+### Introduction of Verified Routes
```
Comment (SteffenDE) [inline CHANGELOG.md:7]:
```suggestion
## Introduction of Verified Routes
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ This feature replaces the `Helpers` module generated in your Phoenix router, but
 will continue to work and be generated. You can disable router helpers by passing the
 `helpers: false` option to `use Phoenix.Router`.
 
-## phx.new revamp
+### phx.new revamp
```
Comment (SteffenDE) [inline CHANGELOG.md:38]:
```suggestion
## phx.new revamp
```

**Comment by SteffenDE** [comment]
Thank you! 🙌🏻

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**